ui/src/main/javascript/app/lobby/lobby.less

```diff
@@ -1,28 +1,71 @@
-@box: black;
-@form: transparent;
-@button: #B07615;
-@font-family_2: Raleway;
-@label: white;
+@import "/style/global.less";
 
-div.create-game-section{
+div.lobby-game-section{
+    height: 100%;
+    padding-top: 5%;
     text-align: center;
-    font-color: white;
+    width: 100%;
+    & .box {
+        box-sizing: border-box;
+        display: inline-block;
+        height: 70%;
+        width: 70%;
+      }
+    & form.sw-align-form{
+        text-align: left;
+      }
+    & .sw-box-1{
+        margin:0 20px 0;
+        width:30%; float:left; 
+      }
+    & .sw-box-2{
+        margin:0 30px 0; 
+        width: 20%; float: left; 
+      }
 
-    &.box {
-       	height: 56%;
-    	width: 40%;
-    	left: 28%;
-    	top: 20%;
-    }
-    & .sw-players {
-          border: solid 2px #dcdcdc;
-          border-radius: 12px;
-          background: @form;
-          font-size: 16px;
-          font-family: @font-family_2;
-          font-color: white;
-          height: 34px;
-          width: 100px;
-          text-align: center;
-      }
-}
+    & .sw-box-3{
+        margin-top: 25px;
+        text-align: bottom;
+        width:20%; float: right;
+      }
+
+    & label.sw-label{
+        display: block;
+        margin: 0.5em 0 0 0;
```

Comment:
Change for
`margin: 0.5em 0 0`;
Example
margin: 25px 50px 75px;
    top margin is 25px
    right and left margins are 50px
    bottom margin is 75px